hawkbit-dmf-amqp/src/main/java/org/eclipse/hawkbit/amqp/AmqpConfiguration.java

```diff
@@ -61,6 +69,7 @@
 
     @Configuration
     @ConditionalOnMissingBean(ConnectionFactory.class)
+    @ConditionalOnProperty(prefix = "hawkbit.dmf.rabbitmq", name = "enabled", matchIfMissing = true)
```

Comment:
I haven't tried that, but I think it's not necessary to use `@ConditionalOnProperty` on the inner class configuration because the outer configuration already has the same conditional, maybe I'm wrong but I thought it's not necessary.